public/Get-DbaRepPublication.ps1

```diff
@@ -1,120 +0,0 @@
-function Get-DbaRepPublication {
```

Comment:
I'd prefer the git history be corrected where we have a rename, and then the change in code. This makes it look like we removed the command.

Reply:
I believe only steps needed to accomplish this...

1. Copy out the current version of the renamed file (one you have currently)
2. Delete that renamed file
3. Checkout this file (old name) from dev branch to reset
4. Rename the file (e.g., `git mv` or Explorer method)
5. Commit that change (this keeps history intact after the rename)
6. Then add your code version back to the file.